upgrade/runner.py

```diff
@@ -70,7 +70,8 @@ def setup_products_for_upgrade(product, os_version):
         create_setup_dict(setups_dict)
         if os.environ.get('RUN_EXISTANCE_TESTS', 'false').lower() == 'true':
             logger.info('Setting up preupgrade datastore for existance tests')
-            set_datastore('preupgrade')
+            set_datastore('preupgrade', 'cli')
+            set_datastore('preupgrade', 'api')
```

Comment:
if i am not wrong , this will create two files with same name in same dir and will overwrite the older file

Reply:
nop .... It will create ```preupgrade_cli``` and ```preupgrade_api```. Same for postupgrade. Check code here -> https://github.com/SatelliteQE/satellite6-upgrade/blob/master/upgrade_tests/helpers/existence.py#L208